var/spack/repos/builtin/packages/qt/package.py

```diff
@@ -338,8 +338,6 @@ class Qt(Package):
             msg="Apple Silicon requires a very new version of qt",
         )
 
-    use_xcode = True
```

Comment:
@sethrj I found that the step where we copy Xcode and modify it to use Spack's compiler wrappers wasn't working for me. I get a prompt asking me to install Xcode command line tools even though they're already installed. If I comment this out, the Qt build succeeds, but Qt crashes at runtime because it can't find libharfbuzz (probably because of the lack of compiler wrappers). Ever seen this before? Any ideas on how we could fix this?

Reply:
@tgamblin @davydden @mathstuf I think all of you have touched the `use_xcode` logic in the past. Any idea what might have changed on newer versions of Xcode (or arm64)?

Reply:
Exact error message:
```
xcode-select: Failed to locate 'make', requesting installation of command line developer tools.
```
But this command is installed on the OS, so I'm not sure why it can't be found.

Reply:
`/usr/bin/make` probaby uses `xcrun` to find an Xcode and run that. `use_xcode` tries to make a "fake" Xcode that spack can reliably inject its compiler wrapper into. Note that with Qt6, this might not be necessary anymore as it uses CMake.

Reply:
Yep, Qt6 doesn't need it. How do I get older Qt working?

Reply:
An issue to that effect would be nice…not sure how it's supposed to be fixed without letting anyone know ;) . (While I did do the fake Xcode thing, I don't use Spack on macOS beyond testing that code, so…)

Reply:
Done: #34064